schematic/utils/df_utils.py

```diff
@@ -31,14 +36,14 @@ def load_df(file_path, preserve_raw_input=True, data_model=False, **load_args):
     # Read CSV to df as type specified in kwargs
     org_df = pd.read_csv(file_path, keep_default_na=True, encoding="utf8", **load_args)
 
+    # only trim if not data model csv
+    if not data_model:
```

Comment:
@GiaJordan the change here makes sense to me. But I am also wondering what's the rationale of moving this block of code out of the original if statement?

Reply:
I see. Thanks for the explanation.